staging/src/k8s.io/apiserver/pkg/cel/library/libraries.go

```diff
@@ -29,6 +29,7 @@ var k8sExtensionLibs = []cel.EnvOption{
 	URLs(),
 	Regex(),
 	Lists(),
+	Authz(),
```

Comment:
maybe add a negative test to CRD validation rules to make sure `authorizer` is unaddressable for now

Reply:
Added in `staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/schema/cel/validation_test.go`